tracker/templates/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /all-incidents/?*
+Allow: /all-incidents/?page=*
+
+Sitemap: https://pressfreedomtracker.us/sitemap.xml
```

Comment:
I think there is probably a way to avoid hard-coding the domain, the sitemap name, or both here. This may or may not appeal to you, but if you give the URL a name, such as 

```
path('sitemap.xml', sitemap, name='sitemap'),
```
Then you can write:

```
Sitemap: https://pressfreedomtracker.us/{% url 'sitemap' %}
```
So that it will refer to whichever path we have defined in our URLs file.
* [`url` docs](https://docs.djangoproject.com/en/4.2/ref/request-response/#django.http.HttpRequest.build_absolute_uri)

To get the absolute URL is a little more trouble, which I was surprised by, but it would involve writing a custom template tag, so that we can write

```
Sitemap: {% absolute_url 'sitemap' %}
```

We'd have to add, for example, to `common_tags.py`, something like this:

```py
@register.simple_tag(takes_context=True)
def absolute_url(context, view_name, *args, **kwargs):
    return context['request'].build_absolute_uri(
        reverse(view_name, args=args, kwargs=kwargs)
    )
```

And then we'd have to add `{% load common_tags %}` to the robots.txt template. 
* [`build_absolute_uri` docs](https://docs.djangoproject.com/en/4.2/ref/request-response/#django.http.HttpRequest.build_absolute_uri)
* [`reverse` docs](https://docs.djangoproject.com/en/3.2/ref/urlresolvers/#reverse)

This is not particularly a lot of additional code in either case, but I'll let you be the judge of how enticing either of these additions sounds.

Reply:
I am aware of the name parameter of `path`, I was just like I don't see any scenario where we would want to change the hostname or the sitemap path name (since both of them are standards). But I guess for future proofing, we can remove hardcoding.